emoji-google/src/main/java/com/vanniktech/emoji/google/GoogleEmoji.java

```diff
@@ -44,27 +60,52 @@ public GoogleEmoji(@NonNull final int[] codePoints, final int x, final int y, fi
   }
 
   @NonNull @Override public Drawable getDrawable(final Context context) {
-    if (sheet == null) {
+    CacheKey key = new CacheKey(x, y);
+    Bitmap bitmap = BITMAP_CACHE.get(key);
+    if (bitmap != null)
+      return new BitmapDrawable(context.getResources(), bitmap);
+    Bitmap strip = loadStrip(context);
+    Bitmap cut = Bitmap.createBitmap(strip, 1, y * 66 + 1, 64, 64);
+    BITMAP_CACHE.put(key, cut);
+    return new BitmapDrawable(context.getResources(), cut);
+  }
+
+  private Bitmap loadStrip(final Context context) {
+    Bitmap strip = (Bitmap) STRIP_REFS[x].get();
+    if (strip == null) {
       synchronized (LOCK) {
-        if (sheet == null) {
-          sheet = BitmapFactory.decodeResource(context.getResources(), R.drawable.emoji_google_sheet);
+        strip = (Bitmap) STRIP_REFS[x].get();
+        if (strip == null) {
+          Log.i(getClass().getSimpleName(), "Loading strip " + x);
```

Comment:
do we need this logging here?

Reply:
This can be removed, it's just to get a picture of what's happening while testing the branch.